src/OrchardCore.Modules/OrchardCore.CustomSettings/AdminMenu.cs

```diff
@@ -33,6 +33,8 @@ public Task BuildNavigationAsync(string name, NavigationBuilder builder)
                         .Add(S["Settings"], settings => settings
                             .Add(new LocalizedString(type.DisplayName, type.DisplayName), type.DisplayName.PrefixPosition(), layers => layers
                                 .Action("Index", "Admin", new { area = "OrchardCore.Settings", groupId = type.Name })
+                                .AddClass(type.Name)
+                                .Id(type.Name)
```

Comment:
Use `HtmlClassify()` as Sebastien mentioned under https://github.com/OrchardCMS/OrchardCore/issues/10367. And there's actually two of that; so while we're at it, please also remove the one in `OrchardCore.ContentManagement.Utilities` because nothing seems to use it, but please check.

Reply:
Seems Seb mentioned `Classified` which is I don't find ;) that's why I used type name instead

Anyhow I will try to react to your suggestion ASAP